src/components/document/document-dnd-context.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import {
+  DndContext, DragEndEvent, DragStartEvent, PointerSensor, useSensor, useSensors
+} from "@dnd-kit/core";
+
+import { useUIStore } from "../../hooks/use-stores";
+
+interface IDocumentDndContextProps {
+  children: any;
+}
+export const DocumentDndContext = ({ children }: IDocumentDndContextProps) => {
+  const ui = useUIStore();
+  const sensors = useSensors(
+    useSensor(PointerSensor, {
+      activationConstraint: {
+        distance: 15
```

Comment:
15 seems like a lot. We're using 3 in CODAP, but something like 5 is pretty reasonable as well.